specification/window.browser.bs

```diff
@@ -0,0 +1,57 @@
+<pre class="metadata">
+Title: window.browser
+Shortname: wecg-browser
+Level: 1
+Group: wecg
+Status: UD
+URL: https://patrickkettner.github.io/window.browser
+Editor: Patrick Kettner, Google, patrickkettner@google.com
+Abstract: This specification reserves the <code>window.browser</code> namespace for use by WebExtensions.
+Repository: patrickkettner/window.browser
+Complain About: accidental-2119 on, missing-example-ids on
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:attribute; text:window
+</pre>
+
+<pre class="anchors">
+spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
+    type: dfn; text: valid non-empty URL; url: valid-non-empty-url-potentially-surrounded-by-spaces
+</pre>
+
+<section class="non-normative">
+  ## Introduction {#introduction}
+  <em>This section is non-normative.</em>
+
+  Modern WebExtensions are an evolution of a system originally introduced in
+  Chrome version 3. In that system, any API that was added was put under the
+  <code>chrome</code> namespace. When WebExtensions were adopted outside of
+  Chromium, other vendors used the more neutral <code>browser</code> namespace.
+  That <code>browser</code> is now considered to canonically correct namespace
+  for all WebExtension APIs.  Some interactions with WebExtensions (such as
+  external messaging from content scripts to background processes) necessitate
+  the exposure of this namespace to the web.
+
+  Given that, this document defines <code>window.browser</code> as reserved for
+  the use of APIs designed to interact with WebExtensions. The specifics of what
+  APIs within <code>window.browser</code> are at present intentionally undefined,
+  and are left up to User Agent implementation.
+</section>
+
+
+  <section>
+    <h3 id="window-interface">
+      <a attribute lt="browser"><code>window.browser</code></a> API
+    </h3>
+
+  {{browser}} is UA defined attribute exposed on {{window}}. When implemented,
```

Comment:
How about extension service workers?

Do we want to reserve `browser` in regular workers as well?

Reply:
added